src/main/java/org/checkstyle/autofix/parser/CheckConfiguration.java

```diff
@@ -21,13 +21,15 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.checkstyle.autofix.CheckstyleModule;
+
 public final class CheckConfiguration {
```

Comment:
We probably need to reconsider the tree structure of the configuration and make it plane per each check that we cover by recipe. This may simplify the rest of the logic related to configuration usage